docs/src/conf.py

```diff
@@ -17,6 +16,17 @@
 
 ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
 sys.path.append(os.path.join(ROOT, "docs", "extensions"))
+sys.path.append(os.path.join(ROOT, "docs", "scripts"))
+
+
+# We use a second (pseudo) sphinx project located in `docs/scipt/gallery` to run the
+# examples and generate the actual outout for our shinx-gallery. This is necessary
+# because here we have to set `METATENSOR_IMPORT_FOR_SPHINX` to TRUE to allow the
+# correct generation of the class and function docstrings which are seperate from the
+# actual code.
+#
+# We the same sphinx gallery confoguartion as in pseudo project
+from gallery.conf import sphinx_gallery_conf  # noqa: E402
```

Comment:
I'd put `sys.path.append(os.path.join(ROOT, "docs", "scripts"))` closer to this line, to indicate they are related